polaris.shopify.com/src/types.ts

```diff
@@ -180,15 +187,14 @@ export enum Breakpoints {
   DesktopLarge = 1600,
 }
 
-export enum StatusName {
-  New = 'New',
-  Deprecated = 'Deprecated',
-  Alpha = 'Alpha',
-  Beta = 'Beta',
-  Information = 'Information',
-  Legacy = 'Legacy',
-  Warning = 'Warning',
-}
```

Comment:
The enum caused all sorts of problems with the types. A union works much better.